src/overmap.h

```diff
@@ -152,6 +152,43 @@ class overmap_special_batch
         point origin_overmap;
 };
 
+static const std::map<std::string, oter_flags> oter_flags_map = {
+    { "KNOWN_DOWN",     known_down     },
+    { "KNOWN_UP",       known_up       },
+    { "RIVER",          river_tile     },
+    { "SIDEWALK",       has_sidewalk   },
+    { "NO_ROTATE",      no_rotate      },
+    { "LINEAR",         line_drawing   },
+    { "SUBWAY",         subway_connection },
+    { "LAKE",           lake },
+    { "LAKE_SHORE",     lake_shore },
+    { "GENERIC_LOOT",   generic_loot },
+    { "RISK_HIGH",      risk_high },
+    { "RISK_LOW",       risk_low },
+    { "SOURCE_AMMO", source_ammo },
+    { "SOURCE_ANIMALS", source_animals },
+    { "SOURCE_BOOKS", source_books },
+    { "SOURCE_CHEMISTRY", source_chemistry },
+    { "SOURCE_CLOTHING", source_clothing },
+    { "SOURCE_CONSTRUCTION", source_construction },
+    { "SOURCE_COOKING", source_cooking },
+    { "SOURCE_DRINK", source_drink },
+    { "SOURCE_ELECTRONICS", source_electronics },
+    { "SOURCE_FABRICATION", source_fabrication },
+    { "SOURCE_FARMING", source_farming },
+    { "SOURCE_FOOD", source_food },
+    { "SOURCE_FORAGE", source_forage },
+    { "SOURCE_FUEL", source_fuel },
+    { "SOURCE_GUN", source_gun },
+    { "SOURCE_LUXURY", source_luxury },
+    { "SOURCE_MEDICINE", source_medicine },
+    { "SOURCE_PEOPLE", source_people },
+    { "SOURCE_SAFETY", source_safety },
+    { "SOURCE_TAILORING", source_tailoring },
+    { "SOURCE_VEHICLES", source_vehicles },
+    { "SOURCE_WEAPON", source_weapon }
```

Comment:
Why is there such strange indentation? First 6 lines have several spaces before the closing bracket, but it didn't indent to anything below it.